docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

```diff
@@ -86,6 +86,38 @@ malicious users.
 
 ====
 
+[[known-issue-938]]
+.Failure in default policy setup may result in policy with no input 
+[%collapsible]
+====
+
+*Details*
+
+During the setup of default policies, an error may occur, such as a network
+error or connectivity issue, that results in a default policy with no input.
+
+*Impact* +
+
+To resolve this problem, you must:
+
+. Delete the default {fleet-server} policy saved object.
+
+. Go to the {fleet} page in {kib} to force reloading of the object.
+
+//QUESTION: Was there a typo in issue 938, or is it really the default
+//Fleet Server policy saved object that needs to be deleted? Or did you mean
+//to say the default agent policy saved objectd?
+//
+//Can you provide steps that describe how the user deletes the
```

Comment:
@nchaulet Did you see the second question? I'm not seeing the saved object on cloud (which is maybe expected). I can spin up a cluster, if necessary, but it would be easier if you could provide instructions that I can dump into this topic.

Reply:
sorry I forget to answer that one it's not possible to remove these saved object from UI or API the easiest way to do it is from the Kibana devtools with that command
```
POST .kibana/_delete_by_query?q=ingest-agent-policies.is_default_fleet_server:true
```